background.js

```diff
@@ -202,3 +202,25 @@ chrome.storage.local.get('version', prefs => {
 		}
 	}
 });
+
+chrome.runtime.onInstalled.addListener(({reason, previousVersion, id} = {}) => {
+	// reason: install, update, chrome_update, shared_module_update
+	if (!/update/.test(reason)) {
+		return;
+  }
+  console.log(reason);
+	const contentScripts = chrome.app.getDetails().content_scripts;
+	chrome.windows.getAll({populate: true}, windows => {
```

Comment:
Well, in case you have 1000+ tabs then it might add like I don't know 100 milliseconds maybe?